com.github.Murmele.scram.yml

```diff
@@ -0,0 +1,32 @@
+app-id: com.github.Murmele.scram
+runtime: org.kde.Sdk
+runtime-version: 5.15-21.08
+sdk: org.kde.Sdk
+command: scram-gui
+rename-icon: scram
+rename-desktop-file: scram-gui.desktop
+rename-appdata-file: scram-gui.appdata.xml
+finish-args:
+- "--socket=wayland"
+- "--socket=fallback-x11"
+- "--filesystem=home"
+- "--share=ipc"
+# - "--device=dri" # TODO: needed?
+modules:
+- boost.json
+# libxml2 is already in the sdk
+- name: scram
+  buildsystem: cmake-ninja
+  config-opts:
+  - -DCMAKE_BUILD_TYPE=Release
```

Comment:
```suggestion
  - -DCMAKE_BUILD_TYPE=RelWithDebInfo
```

Reply:
Do you actually need to set the CMAKE_BUILD_TYPE? iirc flatpak-builder sets that already

Reply:
Not that I can see:

https://github.com/flatpak/flatpak-builder/blob/386976adb8ab97c612394fd12118c7c7e477dda9/src/builder-module.c#L1775

Reply:
changed